### ISSUE TITLE
FOCA needs a SQL database. Please setup your connection and try again.

### ISSUE THREAD

**Issue by alangobetti** [ElevenPaths/FOCA#117]
When i try to launch, after loading i get that:
![image](https://user-images.githubusercontent.com/89802528/131382275-0247f693-e893-4821-8924-e215a3838cae.png)

and that:
![image](https://user-images.githubusercontent.com/89802528/131382436-969638dc-f1e0-4e61-aca6-b7b6bd5d4143.png)

If i try again it backs to the first screen.... ive tryied to replace the adress to localhost as someone here said but nothing change..

**Comment by githubget123** [comment]
Get yourself SQL Server Express (Wiki: https://github.com/ElevenPaths/FOCA/wiki/How-to-set-up-a-SQL-database-connection).
Or use SQLLocalDB (https://www.microsoft.com/en-gb/download/details.aspx?id=42299) 43mb size and install it.
Then from command line run:
- sqllocaldb create yourdbname
- sqllocaldb start yourdbname
Then in the Foca SQL Server details put:
- Server Name: (localdb)\yourdbname
and press Connect.

**Comment by jednoslonce** [comment]
@alangobetti did you solve the problem with connection Foca with SQL database? I have the same problem and I don't know what to do.

**Comment by alangobetti** [comment]
> @alangobetti did you solve the problem with connection Foca with SQL database? I have the same problem and I don't know what to do.

yeah, u just need to download and install SQL (just google it) and it will run

**Comment by freeload101** [comment]
wow no i'm not reading something that can be 3 line powershell...SQL part should be part of install and 1990 called they want there SQL server back if I had time I would look around for new/better option in python / SQLite etc ...

**Comment by xzmshirleyyz2662** [comment]
> Get yourself SQL Server Express (Wiki: https://github.com/ElevenPaths/FOCA/wiki/How-to-set-up-a-SQL-database-connection). Or use SQLLocalDB (https://www.microsoft.com/en-gb/download/details.aspx?id=42299) 43mb size and install it. Then from command line run:
> 
> * sqllocaldb create yourdbname
> * sqllocaldb start yourdbname
>   Then in the Foca SQL Server details put:
> * Server Name: (localdb)\yourdbname
>   and press Connect.

i am so stupid  the first time i downloaded the SQL server without using those commands then i failed when i look back i realize i forget to do the commands then i did and it worked.. thanks so much

**Comment by Th3Gr1ff** [comment]
is ONLY MS SQL types supported? no other non MS types? :/
